data_model/src/query/mod.rs

```diff
@@ -109,10 +109,13 @@ macro_rules! queries {
 pub type QueryId = String;
 
 /// Trait for typesafe query output
-pub trait Query: Into<QueryBox> + seal::Sealed {
+pub trait Query: Into<QueryBox> {
```

Comment:
Can you please the query changes for now? We don't have custom queries yet (and I don't think they'll land before #4569), so this landing this change will just lead to additional conflicts for me (🙁), while not having any benefit (yet)

Reply:
sure